qemu/tests/virtio_fs_share_data.py

```diff
@@ -85,6 +85,7 @@ def get_viofs_exe(session):
     # fio config
     fio_options = params.get('fio_options')
     io_timeout = params.get_numeric('io_timeout')
+    reboot_flag = params.get('reboot_flag', 'no')
```

Comment:
It is recommended to write a new note because this configuration has nothing to do with fio.

Reply:
zhenyzha, hi ,I have modified your part, thanks.